openapi.yaml

```
@@ -8344,6 +8373,97 @@ components:
       required:
         - supply
         - stake
+    network-eras:
+      type: array
+      items:
+        type: object
+        properties:
+          start:
+            type: object
+            description: |
+              Start of the blockchain era,
+              relative to the start of the network
+            properties:
+              time:
+                type: number
+                description: Time in seconds relative to the start time of the network
+              slot:
+                type: integer
+                description: Absolute slot number
+              epoch:
+                type: integer
+                description: Epoch number
+            required:
+              - time
+              - slot
+              - epoch
+          end:
+            type: object
+            description: |
+              End of the blockchain era,
+              relative to the start of the network
+            properties:
+              time:
+                type: number
+                description: Time in seconds relative to the start time of the network
+              slot:
+                type: integer
+                description: Absolute slot number
+              epoch:
+                type: integer
+                description: Epoch number
+            required:
+              - time
+              - slot
+              - epoch
+          parameters:
+            type: object
+            description: Era parameters
+            properties:
+              epoch_length:
+                type: integer
+                description: Epoch length in number of slots
+              slot_length:
+                type: number
```

```suggestion
                type: integer
```
Shouldn't this be integer instead of generic number?

---

It needs to allow floats.